docs/source/data_analysis/HPC-module-SLEAP.md

```diff
@@ -0,0 +1,610 @@
+|                               |                      |                  N/A |
++-------------------------------+----------------------+----------------------+
+
++-----------------------------------------------------------------------------+
+| Processes:                                                                  |
+|  GPU   GI   CI        PID   Type   Process name                  GPU Memory |
+|        ID   ID                                                   Usage      |
+|=============================================================================|
+|  No running processes found                                                 |
++-----------------------------------------------------------------------------+
+```
+Your output should look similar to the above. You will be able to see the GPU
+name, temperature, memory usage, etc. If you see an error message instead,
+(even though you are on a GPU node) please contact the SWC Scientific Computing team.
+
+Next, load the SLEAP module.
+```{code-block} console
+$ module load SLEAP
+Loading SLEAP/2023-08-01
+  Loading requirement: cuda/11.8
+```
+
+To verify that the module was loaded successfully:
+```{code-block} console
+$ module list
+Currently Loaded Modulefiles:
+ 1) SLEAP/2023-08-01
+```
+You can essentially think of the module as a centrally installed conda environment.
+When it is loaded, you should be using a particular Python executable.
+You can verify this by running:
+
+```{code-block} console
+$ which python
+/ceph/apps/ubuntu-20/packages/SLEAP/2023-08-01/bin/python
+```
+
+Finally we will verify that the `sleap` python package can be imported and can
+'see' the GPU. We will mostly just follow the
+[relevant SLEAP instructions](https://sleap.ai/installation.html#testing-that-things-are-working).
+First, start a Python interpreter:
+```{code-block} console
+$ python
+```
+Next, run the following Python commands:
+
+::: {warning}
+The `import sleap` command may take some time to run (more than a minute).
+This is normal. Subsequent imports should be faster.
+:::
+
+```{code-block} pycon
+>>> import sleap
+
+>>> sleap.versions()
+SLEAP: 1.3.1
+TensorFlow: 2.8.4
+Numpy: 1.21.6
+Python: 3.7.12
+OS: Linux-5.4.0-109-generic-x86_64-with-debian-bullseye-sid
+
+>>> sleap.system_summary()
+GPUs: 1/1 available
+  Device: /physical_device:GPU:0
+         Available: True
+        Initialized: False
+     Memory growth: None
+
+>>> import tensorflow as tf
+
+>>> print(tf.config.list_physical_devices('GPU'))
+[PhysicalDevice(name='/physical_device:GPU:0', device_type='GPU')]
+
+>>> tf.constant("Hello world!")
+<tf.Tensor: shape=(), dtype=string, numpy=b'Hello world!'>
+```
+
+If all is as expected, you can exit the Python interpreter, and then exit the GPU node
+```{code-block} pycon
+>>> exit()
+```
+```{code-block} console
+$ exit()
+```
+If you encounter troubles with using the SLEAP module, contact the
+Niko Sirmpilatze of the SWC [Neuroinformatics Unit](https://neuroinformatics.dev/).
```

Comment:
Don't know if you want to link to your email?

Reply:
hmm probably not, people will encounter such problems can reach via the organisational slack